src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/VSResources.resx

```diff
@@ -221,4 +221,7 @@ In order to debug this project, add an executable project to this solution which
   <data name="Renaming_Type_from_0_to_1" xml:space="preserve">
     <value>Renaming Type from '{0}' to '{1}'</value>
   </data>
+  <data name="CannotApplyRename" xml:space="preserve">
+    <value>Cannot apply rename</value>
```

Comment:
Can we look at the error message that legacy project-system provides in this situation and use it?

Reply:
changed to use the message from Roslyn